src/AndersonAcceleration.f90

```diff
@@ -6,53 +6,53 @@
 ! of Michigan and Oak Ridge National Laboratory.  The copyright and license    !
 ! can be found in LICENSE.txt in the head directory of this repository.        !
 !++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
-!> @brief Module provides an interface to Anderson Acceleration
-!>
-!> Currently supported TPLs include:
-!>  - Trilinos
-!>
-!> @par EXAMPLES
-!> @code
-!>
-!> @endcode
-!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+!> @brief A Fortran 2003 module that encompasses the native implementation of
+!>        Anderson Acceleration in Futility.
 MODULE AndersonAccelerationTypes
 USE IntrType
 USE BLAS
-USE trilinos_interfaces
```

Comment:
not a huge fan of dropping the old implementation but agree we need to have a consistent implementation